app_legacy/Helpers/render/layout.php

```diff
@@ -185,6 +185,7 @@ function RenderToolbar(): void
         echo "<ul>";
         echo "<li><a href='/user/$username'>Profile</a></li>";
         echo "<li><a href='/gameList.php?d=$username'>My Sets</a></li>";
+        echo "<li><a href='/individualdevstats.php?u=$username'>My Dev Stats</a></li>";
```

Comment:
Will this appear for all users, including non-devs?

Reply:
The dev stats page will redirect back to the user page if the user doesn't have any achievements created.
https://github.com/RetroAchievements/RAWeb/blob/073ed89421841be34d468ed56e83f346942ea593/public/individualdevstats.php#L21-L26

But I don't want to do that query to populate the menu as it would occur on every page.

I think it's acceptable to hide that link for "retired" devs.

Alternately, the user page relies on the user's ContribCount being non-zero for showing the link. That information might be available here.

Reply:
Added in the checks mentioned above as they make sense (contribution count was easy to pull from the user here), I did leave My Sets and Dev Stats for retired devs since imo even if you are no longer a dev you still have sets and stats but can remove them if still desired